topics/transcriptomics/tutorials/rna-seq-viz-with-cummerbund/tutorial.md

```diff
@@ -0,0 +1,198 @@
+>    > <details>
+>    > <summary>Click to view the answers</summary>
+>    > <ol type="1">
+>    > <li>We need to filter on the column storing the record's significance</li>
+>    > <li>Column 14</li>
+>    > </ol>
+>    > </details>
+>    {: .question}
+{: .hands_on}
+
+# Filtering and sorting
+
+We now want to first highlight the most significant differentially expressed genes in our analysis, and then obtain informative visualizations.
+
+> ### {% icon hands_on %} Hands-on: Extract CuffDiff's most significant differentially expressed genes
+>
+> 1. **Filter** {% icon tool %}: Look up for this tool in the search bar, and select it
+> 2. Select the extracted table as its input, and filter over the target column
+>
+>    > ### {% icon question %} Questions
+>    > 1. What column stores the information of significance for each record?
+>    > 2. Which conditional expression has to be set to filter all records on the selected column?
+>    > 3. What happened to the records in the original table?
+>    >
+>    > <details>
+>    > <summary>Click to view the answers</summary>
+>    > <ol type="1">
+>    > <li>column 14</li>
+>    > <li>c14=='yes'</li>
+>    > <li>All records whose "significant" field was set to "yes" have been retained, while the others filtered out</li>
+>    > </ol>
+>    > </details>
+>    {: .question}
+> 3. **Sort** {% icon tool %}: Look up for this tool in the search bar, and select it
+> 4. Look at your data. The differential expression values are stored on column 10, we will sort (descending) all records on the basis of their value at the 10th column
+> 4. Select the filtered table as the input, and provide the column on which the records have to be sorted, the sorting flavor, and the order
+>
+>    > ### {% icon question %} Questions
+>    > 1. Since the start of our filtering process, how many records now represent the significant subset for extracting informations?
+>    > 2. What does this shrinking of the number of lines represent?
+>    >
+>    > <details>
+>    > <summary>Click to view the answers</summary>
+>    > <ol type="1">
+>    > <li>Click on the boxes in your history, their small preview higlights the number of lines: from ~140,000 to 219</li>
+>    > <li>This process represents a necessary step to gather insights on the biological meaning of our samples in our analyses: putting the original raw RNA-Seq result data into context, cutting down the less-meaningful records to focus on what is needed to go from data to information</li>
+>    > </ol>
+>    > </details>
+>    {: .question}
+{: .hands_on}
+
+# CummeRbund
+
+With CummeRbund we can visualize our RNA-Seq results of interest.
+
+CummeRbund generates always two outputs:
+- the plot
+- the R script responsible for generating the plot
+
+We are interested in visualizing all expression values of all transcripts relative to the most significant differentially expressed gene we found in the previous section.
+
+> ### {% icon hands_on %} Hands-on: Visualization
```

Comment:
I know there is some discussion on whether having screen shots of the tool form with the appropriate options selected/filled in is helpful given that some tool forms are updated fairly frequently and thus can look different from the time the screenshot was taken for the tutorial. Until we come to a consensus on how to best handle tool form updates, it might be helpful to add at least one screenshot to this tutorial of the tool form filled out e.g. for the first Expression Plot. In my experiences, users feel more confident if they have an example completed tool form to compare with their own.

Reply:
Good point. I added the screenshot. I'll remove it in case the consensus decides otherwise

Reply:
I think the current consensus is that we discourage it in general, but that it is a soft rule; as long the parameters are also given in text form (as is the case here) it is up to the topic maintainer (and contributors) to decide if the added value is worth the extra maintenance burden, and for some of the more complex tools I can imagine this may be the case.